.gitignore

```diff
@@ -130,3 +130,10 @@ tests/fsharpqa/testenv/bin/System.ValueTuple.dll
 msbuild.binlog
 /fcs/FSharp.Compiler.Service.netstandard/*.fs
 /fcs/FSharp.Compiler.Service.netstandard/*.fsi
+/FSharp.Core.dll
```

Comment:
Just to aid in traceability, can this be factored into a separate PR?

Reply:
For context, we often end up going back in time to see what else was changed around a particular line of code and get confused by seemingly unrelated changes.

Reply:
This was unintentional, I have rolled back to remove these, good spot